src/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Button = () => {
+  return <button className="btn btn-primary">Search IDs</button>;
```

Comment:
Could we improve this component a bit by making it reusable? Right now if we tried to show this component elsewhere in the app then it would still show "Search IDs". How could we change this component so that it can be used in other parts of our app?